backend/docker/start_fuzz_testing.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# wait for the backend to start up
+# TODO: implement something smarter
+sleep 60
```

Comment:
You could do something like [this](https://github.com/schemathesis/schemathesis/blob/master/.github/workflows/example-manual-build.yml#L62):

```
          timeout 5
          bash -c
          'until printf "" 2>>/dev/null >>/dev/tcp/$0/$1; do sleep 0.2; done'
          localhost 8080
```

Reply:
Thanks, I know and I've seen it too :) I created the PR before the end of my day so others are aware I'm working on it.

Reply:
@lina-is-here yeah, we do something similar in `scripts/ibutsu-pod.sh` although it is external.